src/client/common/experiments/service.ts

```diff
@@ -95,6 +97,15 @@ export class ExperimentService implements IExperimentService {
             return false;
         }
 
+        // If user has .NET interactive installed, we HAVE to be in the native experiment. See this issue:
+        // https://github.com/microsoft/vscode-jupyter/issues/4771
+        if (
```

Comment:
Are the PMs ok with figuring out this via telemetry somehow? I know they were ok with that for AML.

Reply:
Yeah we had a meeting with .NET (Claudia was there) and she's okay with using telemetry to figure out retention.